pkgs/tools/security/truecrack/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, gccStdenv, fetchFromGitLab, cudatoolkit
+, cudaSupport ? false
+, pkg-config }:
+
+gccStdenv.mkDerivation rec {
+  pname = "truecrack";
+  version = "3.6";
+
+  src = fetchFromGitLab {
+    owner = "kalilinux";
+    repo = "packages/truecrack";
+    rev = "debian/${version}+git20150326-0kali1";
+    sha256 = "+Rw9SfaQtO1AJO6UVVDMCo8DT0dYEbv7zX8SI+pHCRQ=";
+  };
+
+  configureFlags = (if cudaSupport then [
+    "--with-cuda=${cudatoolkit}"
+  ] else [
+    "--enable-cpu"
+  ]);
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = lib.optionals cudaSupport [
+    cudatoolkit
+  ];
+
+  cc = "gcc";
+  buildPhase = "make prefix=$out CC=${cc} -j $NIX_BUILD_CORES";
+  installPhase = "make prefix=$out CC=${cc} install";
```

CC should already be defined
```suggestion
   installFlags = [ "prefix=$(out)" ];
   enabledParallelBuilding = true;
```